### DISCUSSION TITLE
Can you change the display of numbers in a chartists graph?

### DISCUSSION THREAD

**Discussion by Andre601** [lowlighter/metrics#970] [Q&A]
Was wondering, if you can change how (and especially *where*) numbers in a chartists type graph (The line graph) are displayed.

Right now one of the SVGs for stragazers looks something like this:
![image](https://user-images.githubusercontent.com/11576465/160141661-b115f277-07ff-4a3f-8a34-d94f594aa810.png)

While it's nice, would I like to put the numbers on the left side of the graph for a more general overview.
Mockup-example:
![image](https://user-images.githubusercontent.com/11576465/160142693-3b9616a6-22b1-492e-95ab-a9da9acd6500.png)

Is there already an option for this?

**Comment by lowlighter** [comment]
Not yet, at least not without directly editing the code.
But maybe we could add a new option to specify raw chartist options that we could pass to the renderer
